exporters/nominatim/nominatim_exporter

```diff
@@ -142,6 +145,10 @@ server.mount_proc "/metrics" do |_request, response|
   response.body << "# HELP nominatim_replication_delay Replication delay in seconds\n";
   response.body << "# TYPE nominatim_replication_delay gauge\n";
   response.body << "nominatim_replication_delay #{replication_delay}\n";
+
+  response.body << "# HELP nominatim_wait_duration_seconds Time spent waiting for a DB connection during a request in seconds\n";
+  response.body << "# TYPE nominatim_wait_duration_seconds gauge\n";
```

Comment:
I think this is a `counter` not a `gauge`?

Reply:
Right, changed.